### DISCUSSION TITLE
error: query: invalid field at position

### DISCUSSION THREAD

**Discussion by demizer** [nvim-treesitter/nvim-treesitter#3510] [Q&A] [answered]
Hello!

I'm working on writing a treesitter parser (in nvim) for a new language called Jakt. I am getting the following error and can't find any info on  best practices for debugging.

From `checkhealth nvim-treesitter` (the error makes the buffer text unreadable...)

```
## The following errors have been detected:
  - ERROR: jakt(highlights): /usr/share/nvim/runtime/lua/vim/treesitter/query.lua:174: query: invalid field at position 1213
    jakt(highlights) is concatenated from the following files:
    | [ERROR]:"/home/demizer/.config/nvim/queries/jakt/highlights.scm", failed to load: /usr/share/nvim/runtime/lua/vim/treesitter/query.lua:174: query: invalid field at position 1213
```
I assume the reason there are 1213 lines is because all off the queries are being concatenated together.

My question:

What is the best practices for turning (line 1213) into the actual line number in my highligts file causing trouble?

Thanks!

**Comment by demizer** [comment]
Is there a place where I can view this concatenated query file?

**Reply by theHamsta** [reply]
I'd recommend to use the query linter GitHub.com/nvim-treesitter/playground. It's possible that you need to change the ft to `query` to enable it

**Reply by demizer** [reply]
Thanks. I was able to resolve the problem for now.

**Comment by demizer** [comment, marked as answer]
Solved by using the query editor to find the bugged field.
